tests/Feature/MeetingTest.php

```diff
@@ -0,0 +1,209 @@
+                ]
+            )
+        );
+
+        // search for partial name
+        $response
+            ->assertStatus(200)
+            ->assertJson([
+                'success' => true,
+                'rowCount' => 1
+            ]);
+
+        $response = $this->actingAs($user, 'api')->getJson(
+            route('api.meeting.list',
+            [
+                'name' => 'Fake Meeting'
+            ])
+        );
+
+        $response
+            ->assertStatus(200)
+            ->assertJson([
+                'success' => true,
+                'rowCount' => 1
+            ]);
+    }
+
+    public function test_search_notFound()
+    {
+        $organizer = factory(User::class)->create();
+        $user = factory(User::class)->create();
+
+        $meeting = factory(Meeting::class)->make();
+        $organizer->meetings()->save($meeting);
+
+        $response = $this->actingAs($user, 'api')->getJson(
+            route(
+                'api.meeting.list',
+                [
+                    'name' => 'Wrong Meeting Name'
+                ]
+            )
+        );
+
+        $response
+            ->assertStatus(200)
+            ->assertJson([
+                'success' => true,
+                'rowCount' => 0
+            ]);
+    }
+
+    public function test_update()
+    {
+        $organizer = factory(User::class)->create();
+
+        $meeting = factory(Meeting::class)->make();
+        $organizer->meetings()->save($meeting);
+
+        $response = $this->actingAs($organizer, 'api')->putJson(
+            route('api.meeting.update', ['meeting' => $meeting->getKey()]),
+            [
+                'name' => 'Fake Meeting'
+            ]
+        );
+
+        $response
+            ->assertStatus(200)
+            ->assertJson([
+                'success' => true
+            ]);
+    }
+
+    public function test_update_onlyByOrganizer()
+    {
+        $organizer = factory(User::class)->create();
+        $user = factory(User::class)->create();
+
+        $meeting = factory(Meeting::class)->make();
+        $organizer->meetings()->save($meeting);
+
+        $response = $this->actingAs($user, 'api')->putJson(
+            route('api.meeting.update', ['meeting' => $meeting->getKey()]),
+            [
+                'name' => 'Fake Meeting'
+            ]
+        );
+
+        $response
+            ->assertStatus(403)
+            /*->assertJson([
+                'success' => false
+            ])*/;
+    }
+
+    public function test_delete()
+    {
+        $organizer = factory(User::class)->create();
+
+        $meeting = factory(Meeting::class)->make();
+        $organizer->meetings()->save($meeting);
+
+        $response = $this->actingAs($organizer, 'api')->deleteJson(
+            route('api.meeting.delete', ['meeting' => $meeting->getKey()])
+        );
+
+        $response
+            ->assertStatus(200)
+            ->assertJson([
+                'success' => true
+            ]);
```

Comment:
Can we also assert there are no meetings? `$this->assertEquals(0, Meeting::count())`